js/services/Vocabulary.js

```diff
@@ -159,11 +152,16 @@ define(function (require, exports) {
 			method: 'POST',
 			contentType: 'application/json',
 			error: authAPI.handleAccessDenied,
-	});
+		});
 
 		return getMappedConceptsByIdPromise;
 	}
 
+	async function  getRecommendedConceptsById(identifiers, url, sourceKey) {
```

Comment:
no point put 'async' here without 'await' in tthe function

Reply:
I'm sorry i accidently clicked resolve conversation, but to your comment about 'if the function is async, there shoudl be an await',

This is not true, and the only thing about async that you should expect is that the function returns a promise. It's not true that the JS interpreter just 'ignores' async when there is no await:  if the function doesn't return a promise, then the function will wrap a promise around the 'raw' value so that the async function returns a promise.

See this for details: https://javascript.info/async-await.

Because this function returns a promise, and I want developers to know that this function returns a promise, I'm leaving this function as 'async'.